DigitalLearningSolutions.Web/Views/CookieConsent/CookiePolicy.cshtml

```diff
@@ -18,7 +29,7 @@
                     <p>We'll only use these cookies if you say it's OK. We'll use a cookie to save your settings.</p>
 
                     <h3>Tell us if we can use analytics cookies</h3>
-                    <fieldset class="nhsuk-fieldset" id="cookie-statistics-form" aria-describedby="cookie-statistics">
+                    <fieldset class="nhsuk-fieldset" id="cookie-statistics" aria-describedby="cookie-statistics">

```

Comment:
To tackle the error thrown by wav plugin, you have used a different ID as the existing one was not valid. What's the source/where did you get this new ID from? Are we sure about this accessibility change?

Reply:
The wave plugin checks for the validity of attributes and in this case the aria-describedby has no referenced element in the html. Hence it was throwing an error. The id is neither used in the js nor it is used to prepare client side script on server side. I did not get this id from anywhere but just making the already existing html correct. I'm not sure about the source you are asking. If you still have doubts then please let me know what do you mean by source and I'll look into it.

Reply:
The id used currently should not be used here. It currently means that it is describing itself (aria-describedby is same as id). 
In order to solve the wave error, we should either try and remove the fieldset altogether OR try to add the id in the h3 heading (tell us.......cookies) and use that id in the fieldset.

Reply:
This is addressed.